homeassistant/components/mobile_app/webhook.py

```diff
@@ -484,3 +484,14 @@ async def webhook_get_config(hass, config_entry, data):
         pass
 
     return webhook_response(resp, registration=config_entry.data)
+
+
+@WEBHOOK_COMMANDS.register("get_panels")
+async def webhook_get_panels(hass, config_entry, data):
+    """Handle a get panels webhook."""
+    panels_data = hass.data[DATA_PANELS]
+    panels = []
+    for panel in panels_data:
+        panels.append(panels_data[panel].to_response())
```

Comment:
We shouldn't do this because:
 - `DATA_PANELS` is owned by the frontend, we're not allowed to access panels from other integrations
 - There is already a websocket command provided by the frontend that the mobile app can use to get the panels

Reply:
It's time for the mobile app to be able to get a helper that does:

 - open websocket connection
 - auth
 - issue a command
 - get a response
 - close connection

Reply:
Yeah was surprising to me that the app doesn't use the WebSocket. I actually had to use the WebSocket to understand panels a bit more before adding this feature. :smiley: 

OK, I'll close this up and look into adding the WebSocket to the android app later/tomorrow. (unless someone beats me to it of course :smiley_cat: )